### PR TITLE
Modify requirements.md

### PR DESCRIPTION
**Pull request by lesser1**
Remove IO patterns requ. after clearification
what was actually meant by that.
Add chaos testing requ.
Clearify description of label Ulm for external devs.

This had to be done due to feedback from supervisors and 
a more detailed answer / requ. from the examiner.

### CODE REVIEW

**Review by KiLanger — APPROVED** [review]

File: docs/requirements/requirements.md [hunk 1]
```diff
@@ -24,7 +24,7 @@ This document is in the making since we are still in the requirements engineerin
 - Researchers:
   - Researcher *Toni*
 
-- External developers *Ulm*
+- *Ulm* -- a collective of developers of a similar system
```
Comment (nk-coding) [inline docs/requirements/requirements.md]: I don't think that fits it too well.
They are rather doing performance / metric experiments and thus need to adapt an existing system

File: docs/requirements/requirements.md [hunk 2]
```diff
@@ -333,6 +342,13 @@ See also R-U-003.
 
 - What other variants could be interesting?
 
+#### R-Conf-006: Configurable reliability
+
+- The system should be configurable regarding its services' reliability.
+- This would be "nice" for testing with chaos testing tools.
```
Comment (nk-coding) [inline docs/requirements/requirements.md]: Doesn't the chaos toolkit simulate unreliable services by randomly killing them / doing other stuff?
Why does our system then need to be configurable too regarding service reliability (besides using the chaos toolkit)?

Comment (nk-coding) [reply docs/requirements/requirements.md]: or what do you exactly mean with "configurable reliability"?

File: docs/requirements/requirements.md (hunk 1 above)
Comment (lesser1) [reply docs/requirements/requirements.md]: How about "experienced users of similar systems; they do metric based performance experiments and regularly have to adapt the underlying reference architecture"?

File: docs/requirements/requirements.md (hunk 2 above)
Comment (lesser1) [reply docs/requirements/requirements.md]: tbh Idk exactly how this toolkit works, I just took another look at the protocol with the examiner and it said "configureable reliability would be nice for tests; check if one can use toolkit Simian army, chaos toolkit". That's where I got this from. 

So we actually only need the requ. R-SArt-005 for chaos testing and I can discard R-Conf-006? Would be fine for me. I could add to R-SArt-005 something like "either the toolkit simulates unreliability or the system must enable this configuration itself"

File: docs/requirements/requirements.md (hunk 1 above)
Comment (nk-coding) [reply docs/requirements/requirements.md]: Sounds good to me

File: docs/requirements/requirements.md (hunk 2 above)
Comment (nk-coding) [reply docs/requirements/requirements.md]: Yeah, merging into the other requirement sounds good

**Review by nk-coding — APPROVED** [review]

**Pull request merged**